default.nix

```diff
@@ -43,7 +43,9 @@ let
       };
     };
 
-  src = pkgs.lib.sources.sourceFilesBySuffices (pkgs.lib.sources.cleanSource ./.) [".c" ".h" ".gif" "Makefile" ".sh" ".json" ".bats" ".txt" ".der"];
+    src = let glyphsFilter = (p: _: let p' = baseNameOf p; in p' != "glyphs.c" && p' != "glyphs.h");
```

Comment:
This could probably use `builtins.elem p' [ "glyphs.c" "glyphs.h" ]` but this is fine with only two.